src/documents/TextToSpeechAPI.md

```diff
@@ -449,15 +457,14 @@ fluid.textToSpeech({
     </tbody>
 </table>
 
-### voiceURI ###
-
+### voice ###
+<!-- See the errata at https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi-errata.html - SpeechSynthesisUtterance must be passed a SpeechSynthesisVoice object to configure the voice under the latest API -->
```

Comment:
I think we should either remove this HTML comment or add it as a note that is visible to the reader.